proxy_http.go

```diff
@@ -260,7 +288,7 @@ func copyHeadersForForwarding(dst, src http.Header) {
 			// section 14.10 of rfc2616
 			// the slice is short typically, don't bother sort it to speed up lookup
 			extraHopByHopHeaders = vv
-		case "Keep-Alive":
+		// case "Keep-Alive":
```

Comment:
After testing, it looks like there's no reason for this to be commented out, so I reverted it to its original form.